lib/common-types.ts

```diff
@@ -714,7 +718,7 @@ export type MRActions = {
     }
     delete: { params: GetAppInstallationParams; return: any }
     getForOrganization: {
-      params: GetOrganizationParams & { appDefinitionId: string }
+      params: GetOrganizationParams & { appDefinitionId: string; spaceId?: string }
```

Comment:
I'm wondering if the space id belongs to a query object 🤔 ?

Reply:
It was just a question - especially thinking about extending it in the future with features like pagination.

Reply:
Wondering if we can document somehow that this is a "filter"?

Reply:
> Wondering if we can document somehow that this is a "filter"?

How & where would you like to document it?

Reply:
Update: There was an issue due to default spaces and I had to put it in a query object after all, also adjusted the JSDoc.